examples/map/map_contouring.py

```diff
@@ -18,7 +19,7 @@
 
 ###############################################################################
 # Now find the contours
-contours = aiamap.contour(50000)
+contours = aiamap.contour(50000 * u.ct)
```

Comment:
this example should also explicitly show users how to find the units of the data - i.e. 


'we can inspect the units of the data by doing:'
``` print(aiamap.units)```